.github/workflows/linux-publish.yml

```diff
@@ -0,0 +1,172 @@
+name: Publish Linux Packages
+
+on:
+  push:
+    branches: [ "main" ]
+    tags: 
+      # Publish semver tags as releases.
+      - app-ziti-console-v*.*.*
+
+jobs:
+  parse-version:
+    name: Find Console Version
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.parse.outputs.version }}
+    steps:
+      - name: Checkout Workspace
+        uses: actions/checkout@v4
+
+      - name: Derive version from tag or package
+        id: parse
+        shell: bash
+        env:
+          GITHUB_REF_NAME: ${{ github.ref_name }}
+          GITHUB_RUN_ID: ${{ github.run_id }}
+        run: |
+          set -o pipefail
+          if [[ "${GITHUB_REF_NAME}" == main ]]
+          then
+            if PACKAGE_VERSION=$(jq -r .version package.json)
+            then
+              echo "version=${PACKAGE_VERSION#v}-${GITHUB_RUN_ID}" | tee -a $GITHUB_OUTPUT
+            else
+              echo "ERROR: Could not find version in package.json"
+              exit 1
+            fi
+          else
+            echo "version=${GITHUB_REF_NAME#app-ziti-console-v}" | tee -a $GITHUB_OUTPUT
+          fi
+
+  build-and-upload-app-ziti-console:
+    needs: parse-version
+    name: Build Console ZIP
+    runs-on: ubuntu-latest
+    env:
+      BUILD_CONFIG: "Release"
+    steps:
+      - name: Checkout Workspace
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          registry-url: https://npm.pkg.github.com/
+
+      - name: build app-ziti-console
+        shell: bash
+        run: ./build.sh /zac/
+
+      - name: upload app-ziti-console
+        uses: actions/upload-artifact@v4
+        with:
+          name: app-ziti-console
+          path: ./dist/app-ziti-console/
+
+  publish-linux-packages:
+    needs:
+      - parse-version
+      - build-and-upload-app-ziti-console
+    name: ${{ matrix.package_name }} ${{ matrix.arch.rpm }} ${{ matrix.nfpm_packager }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package_name:
+          - openziti-console
+        arch:
+          - goreleaser: amd64
```

Comment:
goreleaser? there's nothing 'go' about this project? i'm confused to see 'go' anything in here?